Source/WSTagsField.swift

```diff
@@ -276,8 +276,9 @@ public class WSTagsField: UIView {
     }
     
     public func acceptCurrentTextAsTag(){
-        self.addTag(self.textField.text!)
-        tokenizeTextFieldText()
+        if self.textField.text != ""{
```

Comment:
Try to avoid implicitly unwrapped optionals. Heres why: http://krakendev.io/blog/when-to-use-implicitly-unwrapped-optionals

> You should only ever mark a variable as an Implicitly Unwrapped Optional if you can guarantee that it will NOT be nil at the time it's called.

You can do it more safely like:
```swift
if let currentText = tokenizeTextFieldText() where (self.textField.text?.isEmpty ?? true) == false {
   self.addTag(currentText)
```